types/react-big-calendar/lib/addons/dragAndDrop.d.ts

```diff
@@ -17,8 +17,19 @@ export interface withDragAndDropProps<TEvent extends object = Event> {
   step?: number;
 }
 
+type DragAndDropCalendarProps<TEvent extends object = Event, TResource extends object = object> = CalendarProps<
+    TEvent,
+    TResource
+> &
+    withDragAndDropProps<TEvent>;
```

Comment:
```suggestion
interface DragAndDropCalendarProps<TEvent extends object = Event, TResource extends object = object>
  extends CalendarProps<TEvent, TResource>, withDragAndDropProps<TEvent> {}
```

Reply:
Thanks @DanielRosenwasser, changed this by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46226/commits/948614d0cf124574d506ac1f8613712b8817b892